app/assets/javascripts/.eslintrc.js

```diff
@@ -25,5 +25,12 @@ module.exports = {
     initializeAllFollowButts: false,
     initializeSponsorshipVisibility: false,
     ActiveXObject: false,
+    checkUserLoggedIn: false,
+    sendHapticMessage: false,
+    slideSidebar: false,
+    InstantClick: false,
+    instantClick: false,
+    swipeState: false,
+    time: false,
```

Comment:
I'm conflicted about this, because basically all functions are global in this set of the code, so it's probably better to declare in each file explicitly which global functions are used with `/* global functionName */` than to fill this file with all of them. 

But it's not harmfum anyway, it's also a good way to see in one glance which functions are globally used everywhere

Reply:
I'm with the same doubt, there's a default approach on the project?

Reply:
Not exactly "official policy", but I think declaring `/* global checkUserLoggedIn, InstantClick */` (replacing the names with the ones needed) at the top of the file while we're in this transition is fine :)